src/vm/methodtable.h

```diff
@@ -1982,30 +1982,14 @@ class MethodTable
     //-------------------------------------------------------------------
     // CASTING
     // 
-    // There are two variants of each of these methods:
-    //
-    // CanCastToX
-    // - restore encoded pointers on demand
-    // - might throw, might trigger GC
-    // - return type is boolean (FALSE = cannot cast, TRUE = can cast)
-    //
-    // CanCastToXNoGC
-    // - do not restore encoded pointers on demand
-    // - does not throw, does not trigger GC
-    // - return type is three-valued (CanCast, CannotCast, MaybeCast)
-    // - MaybeCast indicates that the test tripped on an encoded pointer
-    //   so the caller should now call CanCastToXRestoring if it cares
-    // 
     BOOL CanCastToInterface(MethodTable *pTargetMT, TypeHandlePairList *pVisited = NULL);
     BOOL CanCastToClass(MethodTable *pTargetMT, TypeHandlePairList *pVisited = NULL);
-    BOOL CanCastToClassOrInterface(MethodTable *pTargetMT, TypeHandlePairList *pVisited);
-    BOOL CanCastByVarianceToInterfaceOrDelegate(MethodTable *pTargetMT, TypeHandlePairList *pVisited);
-
-    BOOL CanCastToNonVariantInterface(MethodTable *pTargetMT);
+    BOOL CanCastToClassOrInterface(MethodTable *pTargetMT, TypeHandlePairList *pVisited);  
+    BOOL MethodTable::ArraySupportsBizarreInterface(MethodTable* pInterfaceMT, TypeHandlePairList* pVisited);
```

Comment:
```suggestion
    BOOL ArraySupportsBizarreInterface(MethodTable* pInterfaceMT, TypeHandlePairList* pVisited);
```

Some compilers will product a warning on this.